### PR TITLE
Rename Loyalty Implants

### PR DESCRIPTION
**Pull request by flappybatpal**
[QOL][WIKI]

## About the PR 
Renames Loyalty implants to Counter-Revolutionary implants.
Adjusts some text to avoid the word Loyalty.

## Why's this needed? 
Admin discussion indicated it's not unusual for new players to think these are similar to mindslave implants or would work on antagonists. Loyalty is actually used in mindslave text which does not help the confusion.

## Changelog

```changelog
(u)Flappybat
(+)Renamed Loyalty implants to Counter-Revolutionary implants to make their purpose clearer.
```

### CODE REVIEW

**Comment by ThatFiveGuys** [comment]
Love the idea, however i feel as if the name could be better. This may just be a im used to the word loyalty thing and is entirely subjective but i feel as if saying someone is “antirev implanted/antirevved” just sounds less good then saying someone is “loyaltied”

**Comment by Connie-Dun** [comment]
Might I suggest "Nanotrasen Implants"? Still gets across the anti-revolutionary aspect without sounding like a mindslave imo, and can be easily shortened to being "Nano'd";
"Ordering nanos at cargo", "He's been nano'd", "Nanos at sec", etc.
Nobody uses nano'ing to refer to nanomachines, and I think it's distinct enough from "nanites" especially if it really picks up in usage.

**Comment by BatElite** [comment]
The more historically embedded term would probably be counter-revolutionary, which has the added benefit of making NT-sided crew sound stuffy and boring (unlike those cool and hip revs)

**Comment by ThatFiveGuys** [comment]
contrarev implants?

**Comment by Nihisohel** [comment]
Contrarev sounds cool, would make for a better term to call out wanting loyalty for contrarev but it could also be mistaken for contraceptive, i guess?

File: code/obj/item/implant.dm [hunk 1]
```diff
@@ -469,7 +469,7 @@ THROWING DARTS
 					ticker.mode:remove_revolutionary(H.mind)
 				else
 					if (prob(30))
-						H.show_text("<B>The [src] burns and rattles inside your chest! It's attempting to force your loyalty!</B>", "blue")
+						H.show_text("<B>The [src] burns and rattles inside your chest! It's attempting to prevent you revolting!</B>", "blue")
```
Comment (frawhst) [inline code/obj/item/implant.dm:472]: I like the original wording more here. From my POV the main issue with the current name is people assuming the implants will convert traitors, etc- if you're a rev, you already know they're bad for you, and we don't really need to worry about confusion

File: code/obj/item/storage/implant.dm [hunk 2]
```diff
@@ -25,10 +25,10 @@
 	/obj/item/implanter)
 
 /obj/item/storage/box/revimp_kit
-	name = "loyalty implant kit"
+	name = "counter-revolutionary implant kit"
 	icon_state = "implant"
-	desc = "A box containing an implanting tool and five tracking implant cases. The implanter can remove the implants from their cases and inject them in a person, forcing them to be loyal to the Captain and crew."
-	spawn_contents = list(/obj/item/implantcase/antirev = 6,\
+	desc = "A box containing an implanting tool and six tracking implant cases. The implanter can remove the implants from their cases and inject them in a person, supressing their will to rebel against the Captain and crew."
```
Comment (frawhst) [inline code/obj/item/storage/implant.dm:30]:
```suggestion
	desc = "A box containing an implanting tool and six Anti-Revolutionary implant cases. The implanter can remove the implants from their cases and inject them in a person, supressing their will to rebel against the Captain and crew."
```
ancient copypaste strikes again

**Review by frawhst — APPROVED** [review]
otherwise, looks good

File: code/obj/item/implant.dm (hunk 1 above)
Comment (flappybatpal) [reply code/obj/item/implant.dm]: Restored with a slight change to add loyalty to heads of staff.

**Review by flrsh — APPROVED** [review]

**Pull request merged**